guides/doc-Administering_Red_Hat_Satellite/topics/Disabling_Weak_Encryption.adoc

```diff
@@ -7,94 +7,33 @@ You might want to change the encryption settings for {Project} depending on the
 [[Disabling_Weak_SSL_2-0_and_3-0_Encryption]]
 === Disabling Weak SSL 2.0 and SSL 3.0 Encryption
 
-If your {Project} fails Nessus scans because of SSL vulnerabilities, or your security infrastructure requires that you disable SSL 2.0 and SSL 3.0, you can edit the `/etc/foreman-installer/custom-hiera.yaml` file to remove weak encryption.
+Apache and Qpid services in {Project} have SSL encryption disabled by default, however, TLS 1.0 and 1.1 encryption is enabled. If your {Project} and {SmartProxy} fail Nessus scans because of encryption vulnerabilities, or your security infrastructure requires that you disable TSL 1.0 and TSL 1.1, complete this procedure on {Project} and {SmartProxy} to remove weak encryption.
```

Comment:
`SSL encryption` is ambiguous. In many places SSL and TLS are used interchangably. I would not mention SSLv2 and SSLv3 at all. In 2020 no product should even think about shipping them IMHO.
```suggestion
Apache and Qpid services in {Project} have TLS 1.0 and 1.1 encryption enabled by default. This procedure tightens the security to only allow TLS 1.2, as required in various places (such as PCI-DSS regulations).
```

Reply:
Done.